benchexec/tablegenerator/react-table/src/tests/__snapshots__/Summary.test.js.snap

```diff
@@ -10570,7 +10570,7 @@ exports[`Render Summary for multi-table-with-columns.diff.html 1`] = `
         <div
          className="cell"
          dangerouslySetInnerHTML={".036"}
-         title="avg: 0.012, max: 0.015, median: 0.015, min: 0.006, stdev: 0.00424"
+         title="avg: 0.0120, max: 0.015, median: 0.015, min: 0.006, stdev: 0.00424"
```

Comment:
The column has three significant digits, that's why the 0 is included. 
(max, median and min don't have a trailing zeroes as they were taken directly from the dataset. 
Should they also be padded with zeroes then @PhilippWendler  ?